deployment/k8s/postgre-pod.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
use https://github.com/kubernetes/charts/tree/master/stable/postgresql

Reply:
On a second thought - should we even give the option to use a postgres in k8s? I mean it would be nice, but etcd is what sm should be using ultimately.

Reply:
I think for now we could allow it so that we can validate our deployment works there.

When we have time we may implement a db layer and “queries” for etcd and swap the postgres from the sm helm installation